Rakefile

```diff
@@ -36,3 +36,35 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  %x{git log --no-merges --pretty=%s stable..$HEAD}.each_line do |commit_summary|
```

Comment:
@underscorgan and I talked about this, but we're gonna look at using some of the [Envrionment Variables provided by Travis](https://docs.travis-ci.com/user/environment-variables/)